Source/Tests/CompareBicepVersion.Tests.ps1

```diff
@@ -0,0 +1,53 @@
+try {
+    $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
+    Import-Module -FullyQualifiedName "$ScriptDirectory\..\Bicep.psd1"
+}
+catch {
+    Throw "Unable to import Bicep module. $_"
+}
+
+InModuleScope Bicep { 
+    Describe 'CompareBicepVersion' {
+        BeforeAll {
+            Mock LatestBicepVersion -ModuleName Bicep {
+                return '0.3.1'
+            }
```

Comment:
LatestBicepVersion has changed name to ListBicepVersions and has a -Latest parameter to get the same result.
```suggestion
            Mock ListBicepVersions -ModuleName Bicep {
                throw "should not happen"
            }

            Mock ListBicepVersions -ModuleName Bicep -ParameterFilter { $Latest.IsPresent }  {
                return '0.3.1'
            }
```